client/src/config.rs

```diff
@@ -0,0 +1,95 @@
+//! Module for client-related configuration and structs
+
+use core::str::FromStr;
+use std::{path::Path, time::Duration};
+
+use derive_more::Display;
+use eyre::Result;
+use iroha_config::{base, base::UnwrapPartial};
+use iroha_crypto::prelude::*;
+use iroha_data_model::{prelude::*, ChainId};
+use iroha_primitives::small::SmallStr;
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use crate::config::user::RootPartial;
+
+pub mod user;
+
+#[allow(missing_docs)]
+pub const DEFAULT_TRANSACTION_TIME_TO_LIVE: Duration = Duration::from_secs(100);
+#[allow(missing_docs)]
+pub const DEFAULT_TRANSACTION_STATUS_TIMEOUT: Duration = Duration::from_secs(15);
+#[allow(missing_docs)]
+pub const DEFAULT_TRANSACTION_NONCE: bool = false;
+
+/// Wrapper over `SmallStr` to provide basic auth login checking
```

Comment:
```suggestion
/// Basic auth login checking
```

don't put implementation details into documentation